### PR TITLE
Update to Grants page

### PR DESCRIPTION
**Pull request by Smokyish**
This PR updates the copy/text content on the Grants page, also adding a couple of links.

First review on content should be done by @mirtsa and I will then ping a dev for a technical review before merging.

### CODE REVIEW

File: src/view/grants.njk [hunk 1]
```diff
@@ -12,69 +12,68 @@
...
             <h2>1. Background</h2>
             <p class="content-large">
-              The Trustlines Foundation pursues the charitable goal of promoting financial and economic inclusion of all people through decentralised peer-to-peer network protocols that serve common accounting. To ensure the pursuit of this goal in the long term, the Trustlines Foundation provides grants to projects that actively contribute to the development and the usage of the Trustlines ecosystem.
+              The Trustlines Foundation pursues the charitable goal of promoting financial and economic inclusion of all people through decentralized peer-to-peer network protocols that serve common accounting. To ensure the pursuit of this goal in the long term, the Trustlines Foundation provides grants to projects that actively contribute to the development and the usage of the Trustlines ecosystem.
             </p>
 
             <h2>2. Eligibility requirements</h2>
             <ul class="content-large">
-              <li><b>Open source software</b>
+              <li><b>Open-source software</b>
```
Comment (Mirtsa) [inline src/view/grants.njk:20]: it's Open Source, not open-source

File: src/view/grants.njk [hunk 2]
```diff
@@ -12,69 +12,68 @@
...
                 <p>
-                  As the Trustlines Foundation values and aims to promote the philosophy of decentralised networks, projects should respectively also be open source in order to be aligned.
+                  As the Trustlines Foundation values and aims to promote the philosophy of decentralized networks, projects should also be open source.
                 </p>
               </li>
               <li><b>Aiding the Trustlines cause</b>
                 <p>
-                  We are offering grants to anyone who is willing to contribute to the growth and development of the Trustlines ecosystem. This condition is kept unspecific as we desire to foster all types of projects as long as they support our mission.
+                  We are offering grants to anyone willing to contribute to the growth and development of the Trustlines ecosystem. This condition remains unspecific as we desire to foster all kinds of projects as long as they support our mission.
```
Comment (Mirtsa) [inline src/view/grants.njk:27]: All ok except kinds, I'd stick to types

File: src/view/grants.njk (hunk 1 above)
Comment (Smokyish) [reply src/view/grants.njk:20]: It is `open-source software`, or `open source` without the software.

File: src/view/grants.njk [hunk 3]
```diff
@@ -12,69 +12,68 @@
...
                 </p>
               </li>
             </ul>
 
             <h2>3. Exemplary projects</h2>
-            <p><i>If you already have the idea that you want to pursue, you can either skip the following part or gather some extra-inspiration.</i></p>
+            <p><i>Suppose you already have an idea that you want to pursue. In that case, you can skip the following part or gather some extra-inspiration.</i></p>
```
Comment (Mirtsa) [inline src/view/grants.njk:33]: 'An idea' ok, the rest wouldn't change

File: src/view/grants.njk [hunk 4]
```diff
@@ -12,69 +12,68 @@
...
+              <li><b>Research</b>, e.g., economics Ph.D. researching credit networks or P2P payment channel networks, especially on the so-called “imbalanced channel” problem</li>
+              <li><b>User experience enhancements</b>, e.g., building a more advanced mobile application or integrating localization features</li>
             </ul>
 
             <h2>4. Our value proposition</h2>
             <h3>Funding</h3>
-            <p>The total size of the grant depends on the scope of your project, your specific requirements and the estimated budget of the project. It will be paid out in TLN.</p>
-            <p>Furthermore, milestones might potentially be defined for each stage of the project during the application process. These milestones can include criteria that will be required to be met in order to release further payouts.</p>
+            <p>The grant's total size depends on your project's scope, your specific requirements, and the estimated budget of the project. It will be paid out in <a href="https://docs.trustlines.network/docs/resources/glossary.html" target="_blank">TLN</a>.</p>
```
Comment (Mirtsa) [inline src/view/grants.njk:47]: TLN link is okay, as for the rest - don't see the point to change it

File: src/view/grants.njk [hunk 5]
```diff
@@ -12,69 +12,68 @@
...
+            <p>Furthermore, milestones might potentially be defined for each stage of the project during the application process. These milestones can include criteria that will be required to be met for further payouts.</p>
             <p>Lastly, further grants will also be available, subject to the progress of your project.</p>
 
             <h3>Mentorship</h3>
-            <p>To ultimately leverage the  potential of your project, we also offer mentorship by providing you access to our network in the Ethereum ecosystem. For example, if you are looking for exchange opportunities, we are happy to use our connections to other community currency projects.</p>
+            <p>To ultimately leverage your project's potential, we also offer mentorship by providing you access to our network in the Ethereum ecosystem. For example, suppose you are looking for exchange opportunities. In that case, we are happy to use our connections to other community currency projects.</p>
```
Comment (Mirtsa) [inline src/view/grants.njk:52]: don't see the point of these changes

File: src/view/grants.njk [hunk 6]
```diff
@@ -12,69 +12,68 @@
...
             <p>Once we receive your application, we will start the evaluation process. The evaluation process includes an initial screening of your application, followed by an in-depth review and potentially also an interview if necessary.</p>
-            <p>The duration of this process is highly dependent on the level of detail of your application and the scope of your project in general. Thus we ask for your understanding that we are not able to provide any specific timeframe in which the evaluation process will take place.</p>
-            <p>Please be aware that we might contact you for additional information throughout the whole process in case we are missing any, so we kindly ask you to be available.</p>
-            <p>You can apply via <a href="https://docs.google.com/forms/d/e/1FAIpQLSdSGkkafn3jDvFmv8S_FwWVoSkGXc6UGMDjTKw6Maoibcz5Jg/viewform">Google Forms.</a></p>
+            <p>This process's duration depends on your application's level of detail and the scope of your project in general. Thus we ask for your understanding that we cannot provide any specific timeframe in which the evaluation process will occur.</p>
```
Comment (Mirtsa) [inline src/view/grants.njk:61]: I'd stick to 'The duration of this process' and 'the level detail of your application'

**Review by Mirtsa — COMMENTED** [review]
Please check out the comments. Everything which is not commented is approved

**Comment by Smokyish** [comment]
@compojoom Please do a technical review of the PR, or assign someone else for a review.

**Review by cducrest — APPROVED** [review]

**Pull request merged**